arkane/gaussian.py

```diff
@@ -497,3 +497,31 @@ def loadNegativeFrequency(self):
         if frequency is None:
             raise Exception('Unable to find imaginary frequency in Gaussian output file {0}'.format(self.path))
         return frequency
+
+    def get_level_of_theory(self):
+        """
+        Returns a string describing the level of theory used in the
```

Comment:
Does this return the method without the basis set?
I think only the method + basis set (e.g., b3lyp/6-311g) is called a level of theory, otherwise let's just call it "method"

Reply:
renamed to get_quantum_method